scripts/plugins/support-pssl.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+#
+# Copyright (c) Citrix Systems 2017. All rights reserved.
+
+set -e
+
+# Open/Close port 6640 when pssl is used in ovs
+
+OP="$1"
+rule="RH-Firewall-1-INPUT -p tcp -m conntrack --ctstate NEW -m tcp --dport 6640 -j ACCEPT"
+
+case "${OP}" in
+    start)
+        if iptables -S | grep -e "${rule}" > /dev/null
+        then
+            echo "port 6640 is opened already"
+            exit 1
```

Comment:
I prefer to not return an error here to make the operation idempotent. The caller is only interested in whether the state if the system is right after executing the command. Whether it was right already makes no difference.